node/middlewares/common/isAdminAuthenticated.ts

```diff
@@ -0,0 +1,29 @@
+import { json } from 'co-body'
+import { CRON } from '../../utils/constants'
+
+export async function isAdminAuthenticated(
+  ctx: Context,
+  next: () => Promise<any>
+) {
+    interface CtxObject {token: string }
+    try {
+
+      const {token}: CtxObject = await json(ctx.req)
+
+    if (token !== CRON.authToken) {
```

Comment:
would be better to read from settings than a constant

Reply:
done